lib/monitoring/aws-elasticache/ElastiCacheClusterMetricFactory.ts

```diff
@@ -112,6 +112,21 @@ export class ElastiCacheClusterMetricFactory {
     );
   }
 
+  /**
+   * @see https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheMetrics.Redis.html
```

Comment:
It also notes:

> 
On an ElastiCache host, background processes monitor the host to provide a managed database experience. These background processes can take up a significant portion of the CPU workload. This is not significant on larger hosts with more than two vCPUs. But it can affect smaller hosts with 2vCPUs or fewer. If you only monitor the `EngineCPUUtilization` metric, you will be unaware of situations where the host is overloaded with both high CPU usage from Redis and high CPU usage from the background monitoring processes. Therefore, we recommend monitoring the `CPUUtilization` metric for hosts with two vCPUs or less.

We should clarify that you may need to monitor both.

Reply:
done (see below)